src/script/components/message.ts

```diff
@@ -169,12 +244,12 @@ class Message {
     amplify.publish(SHOW_LEGAL_HOLD_MODAL, this.conversationRepository.active_conversation());
   };
 
-  showContextMenu(messageEntity, event) {
+  showContextMenu(messageEntity: ContentMessage, event: Event) {
     const entries = [];
 
     if (messageEntity.is_downloadable()) {
       entries.push({
-        click: () => messageEntity.download(),
+        click: () => (messageEntity as any).download(),
```

Comment:
A `ContentMessage` has the `download` property defined?!

Reply:
Whoops, left over from trying to figure out if `message` is of type `Message` or `ContentMessage` :)